subsys/net/openthread/Kconfig.defconfig

```diff
@@ -82,6 +82,9 @@ choice OPENTHREAD_SECURITY
 	default OPENTHREAD_NRF_SECURITY_CHOICE if SOC_FAMILY_NRF
 endchoice
 
+config MBEDTLS_PSA_CRYPTO_EXTERNAL_RNG
+	default n if OPENTHREAD_NRF_SECURITY_PSA
```

Comment:
Could you explain why this is needed?

Reply:
As far as I remember, I added it to my recent PR regarding PSA crypto, because I wanted to use the internal implementation of the RNG when the PSA api is used, but I don't remember why it wasn't setting automatically... Maybe now we can remove those lines.

Reply:
I've removed this part and tested locally- commissioning/joining and COAPS look good. I'll schedule full nightly suite with PSA enabled too.

Reply:
I've checked build files once more and seems like external RNG is enabled by default (here: 
https://github.com/nrfconnect/sdk-nrf/blob/3003f42ee1a14ac6ca6f3cb6990ea2fcc05bb6a7/subsys/nrf_security/Kconfig.psa#L22 )
Do we want to keep it that way then? Or should we switch to internal RNG as in your recent PR?

Reply:
@joerchan What is the recommendation of the crypto team with regards to MBEDTLS_PSA_CRYPTO_EXTERNAL_RNG when PSA crypto API is used? I don't quite understand what it changes under the hood.

Reply:
@Damian-Nordic This is an old configuration, from when we had MBEDTLS as the PSA Core.

It used to be a setting that would direct the RNG to an external source i.e. cc3xx. This is now replaced by regular PSA crypto drivers.

We have removed this configuration, so please remove this here as well.

Reply:
Thanks for the clarification!

Reply:
removed, thanks